src/MainConfigure.cpp

```diff
@@ -29,6 +29,18 @@ int main(int argc, char const* argv[])
     // configuration mode
     driver.setConfigurationMode();
     driver.startAcquisition();
+
```

Comment:
Since this just applies a config file, why starting the acquisition here in the first place?

Reply:
Removed it.